src/test/java/CupTest.java

```diff
@@ -0,0 +1,34 @@
+package test.java;
+
+import main.java.Cup;
+import org.junit.jupiter.api.RepeatedTest;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+public class CupTest {
+
+    @RepeatedTest(20)
+    void cupRollsValue() {
+        Cup cup = new Cup(2);
+        cup.roll();
+        assertTrue(2 <= cup.getTotal() && cup.getTotal() <= 12);
+    }
+
+    @Test
+    void negativeNumberOfDice() {
+        try{
+            Cup cup = new Cup(-1);
+            fail("Exception not thrown");
+        }catch(Exception e){
+            // OK an exception has been thrown
+        }
+    }
```

Comment:
If you want to test if it thow a exception try this :

```suggestion
    @Test
    void negativeNumberOfDice() {
        Assertions.assertThrows(RuntimeException.class, ()->
                Cup cup = new Cup(-1)
        );
    }
```

Reply:
Got it, changing now